hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -525,39 +525,39 @@
     {
       name: chip_sw_uart_tx_rx
       uvm_test_seq: chip_sw_uart_tx_rx_vseq
-      sw_images: ["//sw/device/tests/sim_dv:uart_tx_rx_test:1:new_rules"]
+      sw_images: ["//sw/device/tests:uart0_tx_rx_test:1:new_rules"]
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+uart_idx=0", "+calibrate_usb_clk=1"]
       reseed: 5
     }
     {
       name: chip_sw_uart_tx_rx_idx1
       uvm_test_seq: chip_sw_uart_tx_rx_vseq
-      sw_images: ["//sw/device/tests/sim_dv:uart_tx_rx_test:1:new_rules"]
+      sw_images: ["//sw/device/tests:uart0_tx_rx_test:1:new_rules"]
```

Comment:
Note: I've used uart0 for all four of these even though I could have used `uart1`, `uart2`, etc. to avoid having to compile the program four times. DVSIM backdoors this to change it to the correct UART before it runs.